pyspeckit/spectrum/units.py

```diff
@@ -417,6 +423,11 @@ def __new__(self, xarr, unit="Hz", frame='rest', frame_offset=0.0,
         else:
             subarr.velocity_convention = velocity_convention
 
+        subarr._equivalencies = equivalencies
+        if center_frequency:
+            subarr.center_frequency, subarr._equivalencies  = self.find_equivalencies(velocity_convention, center_frequency, center_frequency_unit, equivalencies)
```

Comment:
Let's call the `center_frequency` something else: `subarr.refX`, for backward compatibility.  We can also add:
`subarr.rest=subarr.refX` because the "rest wavelength" or "rest frequency" is more descriptive.  This is expanding the API a little, but it shouldn't cause any problems.